lib/node_modules/@stdlib/stats/base/range/lib/ndarray.js

```diff
@@ -40,10 +41,13 @@ var isnan = require( '@stdlib/math/base/assert/is-nan' );
 * var x = [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ];
 * var N = floor( x.length / 2 );
 *
-* var v = range( N, x, 2, 1 );
+* var v = range( 4, x, 2, 1 );
 * // returns 6.0
 */
-function range( N, x, stride, offset ) {
+function range( N, x, strideX, offsetX ) {
+	var obj = arraylike2object( x );
```

Comment:
This is not what we want. Please study other implementations. Accessing directly in `data` is precisely what we want to avoid if `x` is an accessor array. That is the entire point of `accessors.js`.